ai-platform-unified/notebooks/official/pipelines/automl_tabular_classification_beans.ipynb

```diff
@@ -0,0 +1,1040 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    def classif_model_eval_metrics(
</pre><p>Not sure if this was intentional or not, but 'classif'?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
It _was_ an intentional abbreviation of 'classification'...  😬